app/controllers/api/base_controller/logger.rb

```diff
@@ -35,7 +35,10 @@ def log_api_request
         @parameter_filter ||= ActionDispatch::Http::ParameterFilter.new(Rails.application.config.filter_parameters)
         return unless api_log_info?
         log_request("Request", @req.to_hash)
-        log_request("Parameters", @parameter_filter.filter(request.query_parameters.merge("body" => @req.json_body)))
+        unfiltered_params = request.query_parameters
+                            .merge("body" => @req.json_body)
+                            .merge(params.permit(:action, :controller, :format).to_h)
```

Comment:
glad to see we could still leverage params for those.   

Can you move that line up one,  I'd like to keep body at the end of that logged line.  Thanks.